src/helpers/calculatieStock.js

```diff
@@ -0,0 +1,10 @@
+import amountSold from "./amountSold.js";
+import quantityPurchase from "./quantityPurchase.js";
+
+function calculatieStock (productArray) {
+    const productPurchase = quantityPurchase(productArray);
+    const productSold = amountSold(productArray);
+    return productPurchase - productSold;
+}
```

Comment:
Dit had korter gekund zonder variabelen. Als je de variabelen verder niet nodig hebt in de applicatie heb je ze dus eigenlijk niet nodig.

return quantityPurchase(productArray) -  amountSold(productArray);